src/screens/ENSAssignRecordsSheet.js

```diff
@@ -56,73 +63,110 @@ export default function ENSAssignRecordsSheet() {
     ],
   });
 
+  const avatarRadius = 35;
+
+  const formIsEmpty = Object.values(values).join('');
```

Comment:
Maybe we could move this logic inside of `useENSProfileForm` and call it `isEmpty`?

```tsx
const {
  isEmpty,
  ...
} = useENSProfileForm(...)
```